### PR TITLE
[release only] Pin Docker images for release.

### PR DESCRIPTION
**Pull request by atalman**
Pins all Docker images used for  building to Sept 19 timeframe (time of final RC build for release 2.1.0)

### CODE REVIEW

File: .github/workflows/docker-release.yml [hunk 1]
```diff
@@ -91,12 +91,9 @@ jobs:
...
           echo "${RUNNER_TEMP}/bin" >> "${GITHUB_PATH}"
-          # Generate PyTorch version to use
-          if [[ ${CHANNEL} == "release" ]]; then
-            echo "PYTORCH_VERSION=2.1.0" >> "${GITHUB_ENV}"
-          else
-            echo "PYTORCH_VERSION=$(python3 .github/scripts/generate_pytorch_version.py)" >> "${GITHUB_ENV}"
-          fi
+          # Generate PyTorch version to use without suffix
+          echo "PYTORCH_VERSION=$(python3 .github/scripts/generate_pytorch_version.py --no-build-suffix)" >> "${GITHUB_ENV}"
+
```
Comment (malfet) [inline .github/workflows/docker-release.yml:96]: This does not look related to the PR title, is it?

Comment (atalman) [reply .github/workflows/docker-release.yml:96]: No this is what the PR say : [release only] 1 - Pin Docker images for release. 2 - Fix docker release build

Comment (atalman) [reply .github/workflows/docker-release.yml]: Done remove docker release build change

**Review by huydhn — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Comment by malfet** [comment]
IMO we should tag those, otherwise they might be recycled after some period of time
